### ISSUE TITLE
could you provide the training loss

### ISSUE THREAD

**Issue by guozhiyao** [Vision-CAIR/MiniGPT-4#167]

**Comment by guozhiyao** [comment]
I change the 13b llm and pretrain the project layer by loading your stage2 checkpoint. The initial loss is around 5 and drop slowly. It is around 2.5 after 1.5m samples. Is it normal? I have try to fine tune your model with the same config and the loss is around 1.4. I do not know if my pretrain loss can match your training log or not.

**Comment by yPanStupidog** [comment]
Can you do fine-tune using 7B?

**Comment by guozhiyao** [comment]
> Can you do fine-tune using 7B?

I try to change the llm. But I do not know if my pretrain loss is normal. Because I found the fine tune loss of your 13b is around 1.4. I have not try the 7b yet.
Could you provide the training log of 7b or 13b please?

**Comment by yPanStupidog** [comment]
Could you please help me check whether finetune could be applied on 7B？I always met errors！

**Comment by yuanlisky** [comment]
13B stage2 finetune
in my case, the initial loss is 3.8, epoch 4, 200 iters, the best loss is 1.9

**Comment by ImKeTT** [comment]
> > Can you do fine-tune using 7B?
> 
> @yPanStupidog I try to change the llm. But I do not know if my pretrain loss is normal. Because I found the fine tune loss of your 13b is around 1.4. I have not try the 7b yet. Could you provide the training log of 7b or 13b please?

a quick question, what's your config for pre-training? I tried to pre-train this model using the default training config (stage 1) with a 7b LLM, it gave me `nan` loss at the beginning, did you meet the similar error? Thanks

**Comment by LIO-H-ZEN** [comment]
> > > Can you do fine-tune using 7B?
> > 
> > 
> > @yPanStupidog I try to change the llm. But I do not know if my pretrain loss is normal. Because I found the fine tune loss of your 13b is around 1.4. I have not try the 7b yet. Could you provide the training log of 7b or 13b please?
> 
> @guozhiyao a quick question, what's your config for pre-training? I tried to pre-train this model using the default training config (stage 1) with a 7b LLM, it gave me `nan` loss at the beginning, did you meet the similar error? Thanks

same error

**Comment by hhllxx1121** [comment]
> > > > Can you do fine-tune using 7B?
> > > 
> > > 
> > > @yPanStupidog I try to change the llm. But I do not know if my pretrain loss is normal. Because I found the fine tune loss of your 13b is around 1.4. I have not try the 7b yet. Could you provide the training log of 7b or 13b please?
> > 
> > @guozhiyao a quick question, what's your config for pre-training? I tried to pre-train this model using the default training config (stage 1) with a 7b LLM, it gave me `nan` loss at the beginning, did you meet the similar error? Thanks
> 
> same error

I have encountered the same problem. Have you resolved it

**Comment by Zhanghahah** [comment]
same issue, could you provide some insights?

**Comment by GDUTT1** [comment]
I use vicuna7b  and EVA01-CLIP-g-14 , the initial loss is 6.8479.  After 1.0M sampls, the loss is 2.6, is it normal？
